python/dgl/contrib/dis_kvstore.py

```diff
@@ -11,179 +11,106 @@
-    close_shared_mem : bool
-        Close local shared-memory tensor access.
-    msg_queue_size : int
-        Size of message queue (2GB by default)
-
-    Returns
-    -------
-    KVClient
-        client handle
-    """
-    assert len(ip_config) > 0, 'ip_config cannot be empty.'
-    assert len(ndata_partition_book) > 0, 'ndata_partition_book cannot be empty.'
-    assert len(edata_partition_book) > 0, 'edata_partition_book cannot be empty.'
-
-    server_namebook = read_ip_config(ip_config)
-
-    client = KVClient(server_namebook=server_namebook, close_shared_mem=close_shared_mem, msg_queue_size=msg_queue_size)
-
-    for name, data in ndata_partition_book.items():
-        client.set_partition_book(name=name, partition_book=data)
-
-    for name, data in edata_partition_book.items():
-        client.set_partition_book(name=name, partition_book=data)
-
-    client.connect()
-
-    print("Client %d (%s) connected to kvstore ..." % (client.get_id(), client.get_addr()))
-
-    return client
-    
-
 class KVServer(object):
     """KVServer is a lightweight key-value store service for DGL distributed training.
 
-    In practice, developers can use KVServer to hold large graph features or graph embeddings 
-    across machines in a distributed setting. User can re-wriite _push_handler and _pull_handler 
-    to support flexibale algorithms.
+    In practice, developers can use KVServer to hold large-scale graph features or 
+    graph embeddings across machines in a distributed setting. User can re-wriite _push_handler() 
+    and _pull_handler() API to support flexibale algorithms.
+
+    DGL kvstore supports backup servers. That means we can lunach many servers on the same machine and all of 
+    these servers share the same shared-memory tensor.
 
     Note that, DO NOT use KVServer in multiple threads on Python because this behavior is not defined.
 
-    For now, KVServer can only run in CPU, and we will support GPU KVServer in the future.
+    For now, KVServer can only run in CPU, we will support GPU KVServer in the future.
 
     Parameters
     ----------
     server_id : int
-        ID of current kvserver node (start from 0).
+        KVServer's ID (start from 0).
     server_addr : str
-        IP address and port of current KVServer node, e.g., '127.0.0.1:50051'.
+        machine ID, IP address and port of KVServer, e.g., '0|127.0.0.1:50051'.
     num_client : int
         Total number of clients connecting to server.
-    msg_queue_size : int
-        Size of message queue (2GB by default)
+    queue_size : int
+        Sise (bytes) of kvstore message queue buffer (~20 GB on default).
     net_type : str
         networking type, e.g., 'socket' (default) or 'mpi' (do not support yet).
     """
-    def __init__(self, server_id, server_addr, num_client, msg_queue_size=2 * 1024 * 1024 * 1024, net_type='socket'):
+    def __init__(self, server_id, server_addr, num_client, queue_size=20000000000, net_type='socket'):
```

Comment:
I perfer 2 * 1024 * 1024 * 1024. It is more straight forward.

Reply:
Sure, I will change that.